stuff/config/colornames.txt

```diff
@@ -114,55 +180,101 @@
   <color name="MediumTurquoise">#48D1CC</color>
   <color name="MediumVioletRed">#C71585</color>
   <color name="MidnightBlue">#191970</color>
+  <color name="Mint">#98FB98</color>
   <color name="MintCream">#F5FFFA</color>
   <color name="MistyRose">#FFE4E1</color>
   <color name="Moccasin">#FFE4B5</color>
+  <color name="Moss">#6C7131</color>
+  <color name="Mulberry">#912185</color>
   <color name="NavajoWhite">#FFDEAD</color>
   <color name="Navy">#000080</color>
+  <color name="Nectar">#F28355</color>
+  <color name="NeonPink">#F770AB</color>
+  <color name="NeonOrange">#F77857</color>
+  <color name="NeonYellow">#FBEE14</color>
+  <color name="Ocean">#016064</color>
+  <color name="Ochre">#CC7722</color>
   <color name="OldLace">#FDF5E6</color>
   <color name="Olive">#808000</color>
-  <color name="OliveDrab">#6B8E23</color>
```

Comment:
It's not clear why you removed this color.
It's fine to have multiple color names that have the same value.
OliveDrab is a recognized colorname whereas OliveDark appears to be a newly invented one.